pkg/distributor/distributor_ring.go

```diff
@@ -16,8 +16,13 @@ import (
 	"github.com/grafana/dskit/kv"
 	"github.com/grafana/dskit/netutil"
 	"github.com/grafana/dskit/ring"
+)
 
-	util_log "github.com/grafana/mimir/pkg/util/log"
+const (
+	// ringNumTokens is how many tokens each distributor should have in the ring.
+	// Distributors use a ring because they need to know how many distributors there
+	// are in total for rate limiting.
+	ringNumTokens = 1
```

Comment:
Even though we use the ring just to know the number of instances, I'd still set this to a sane value, as someone may use the ring for something _decent_ in the future and can oversee this detail.

```suggestion
	ringNumTokens = 128
```

Reply:
More tokens make the ring bigger, it means more memory usage, larger CAS operations (consul) or bigger gossiped messages (memberlist). We use 1 token in this ring for ages (2 years) now. I'd keep it as it until we actually need more tokens.

Reply:
Well, if this isn't new, then ok!